src/components/table/OcTableFiles.vue

```diff
@@ -337,6 +368,33 @@ export default {
     },
   },
   methods: {
+    resetDropPosition(id, event) {
+      const instance = this.$refs[id].tippy
+      this.displayPositionedDropdown(instance, event)
+    },
+
+    showContextMenu(rows, event) {
+      event.preventDefault()
+
+      const instance = rows.$el.getElementsByClassName("oc-table-files-btn-action-dropdown")[0]
```

Comment:
is the element always available or shouldn't we take care with an early return?

Reply:
👍 for early return. just to be sure.

Reply:
Hmmm... thinking about this now, wouldn't it make more sense to do something like this?
```js
const instances = rows.$el.getElementsByClassName("oc-table-files-btn-action-dropdown")
if (instances.length === 0) {
  return
}
```
?

Reply:
And then use `instances[0]` in line 387.

Reply:
Very true yeah, fixing it

Reply:
<img width="1021" alt="Screenshot 2021-07-22 at 12 35 36" src="https://user-images.githubusercontent.com/3532843/126626419-cdc3bf16-9adf-49c5-b1cc-d44143b6be08.png">

I should've read the spec, sorry for the noise. Your previous solution was perfectly fine, and shorter. 🙄